src/main/target/common_hardware.c

```diff
@@ -111,6 +111,17 @@
     BUSDEV_REGISTER_I2C(busdev_qmc5883,     DEVHW_QMC5883,      QMC5883_I2C_BUS,    0x0D,               NONE,           DEVFLAGS_NONE);
 #endif
 
+#if defined(USE_MAG_AK8963)
+    #if defined(AK8963_SPI_BUS)
+    BUSDEV_REGISTER_SPI(busdev_ak8963,      DEVHW_AK8963,       AK8963_SPI_BUS,     AK8963_CS_PIN,      NONE,           DEVFLAGS_NONE);
```

Comment:
Can AK8963 be on native SPI? I though it's only I2C sensor

Reply:
It can be I2C or SPI, The AlienFlightNG Anubis16 is an example for SPI: 

https://www.alienflightng.com/flight-controllers/f7-anubis16/

We have two more FC's in planning with this combination. The HMC5338 is EOL, is not produced anymore and will get unavailable soon. So the AK8963 likely will show up more often.